misaligned/misaligned.js

```diff
@@ -1,16 +1,14 @@
-const {expect} = require('chai')
-
-function print_color_map() {
-    const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
-    const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
-    for (let i = 0; i < majorColors.length; i++) {
-        for (let j = 0; j < minorColors.length; j++) {
-            console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
-        }
-    }
-    return majorColors.length * minorColors.length;
-}
-
-result = print_color_map();
-expect(result).equals(25);
-console.log('All is well (maybe!)');
+const {expect} = require('chai')
+
+function print_color_map() {
+    const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
+    const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
+    for (let i = 0; i < majorColors.length; i++) {
+        for (let j = 0; j < minorColors.length; j++) {
+            console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
```

Comment:
This is not testable in its current form. Separate the calculation and formatting into different functions and write tests for them

Reply:
Okay sir will do